data/coalition/resistance intro.txt

```diff
@@ -0,0 +1,1704 @@
+			label end
+			`	The final torpedo is brought out of your ship, and the Saryds start bringing them inside a cave leading into the mountain. The Saryd who paid you thanks you again, and follows after the others. You head back to your ship and fly it to the hangars, hoping to find something warm to drink.`
+
+mission "Resistance: Heat"
+	landing
+	name "Heated Smuggling"
+	description "Deliver two plasma turrets and ten flamethrowers to <destination>."
+	source
+		attributes saryd
+		not planet "Secret Sky"
+	destination "Secret Sky"
+	to offer
+		has "Resistance: Torpedoes: done"
+		has "event: flamethrower available"
+		random < 25 + 3 * "assisted resistance" * "assisted resistance"
+		not "Heliarch Investigation 2: active"
+		not "Heliarch License: done"
+	to fail
+		has "Heliarch Investigation 2: active"
+	on offer
+		conversation
+			`Once again, the Lunarium's device starts beeping as you land, and your monitor shifts to the same messaging screen as last time. "Hello again Captain. Chiree, it is. Thank you for the torpedoes, I must, already at work on producing more like them, we are. Win us battles, however, they alone will not, so asking you to help us once again, I am. Received word, we have, of some more weapons developed by humans. Quickly heat up ships, they can, and so a great help against Heliarch attacks, they would be."`
+			choice
+				`	"What do you need me to do?"`
+					goto accept
+				`	"You have 'received word' of that? From whom? Who's telling you of these weapons?"`
+			`	For how fast she typed before, Chiree takes a long time to answer. "Know that, we do not," is what first appears on the screen. "First contacted centuries ago, we were. Unsure, we still are, if trustworthy, whoever is sending the messages is, but always right on their information, they are. The first to bring us human outfits, you are not; whoever they are, brought us caches of some human platings and guns, decades ago, they did. Seen, their ships never were, only the crates and containers, we found."`
+			choice
+				`	"So why do you trust these mystery informants so much?"`
+				`	"Whatever the case, just tell me what you need me to do."`
+					goto accept
+			`	"Because helped us, they always have, and in dire need of help, we are," she shoots back in an instant. "Given us a fighting chance, they have," she adds.`
+			label accept
+			`	She quickly types out another message. "A larger delivery, this will be, so a good hiding spot, we'll need. To <destination>, bring two 'plasma turrets' and ten 'flamethrowers' you must. Paid <payment>, you will be."`
+			`	The beeping stops and your monitor returns to normal again.`
+				accept
+	on visit
+		dialog `You've arrived on <planet>, but the ship carrying the plasma turret hasn't arrived in this system yet. Better depart and wait for it.`
+	on complete
+		outfit "Plasma Turret" -2
+		outfit "Flamethrower" -10
```

Comment:
I don't really see a problem with the cargo space, especially since it's not even the largest haul of the chain (that being the Reactors mission).
The player is told in advance what they need to bring, and can figure out how much cargo they need to be able of carrying that into the Coalition, so it's not like this'd block someone who's in their combat flagship with 0 cargo space. At this point one should be able of easily buying at least a Hauler III and swapping their JD to it in order to complete the mission.